### ISSUE TITLE
Filter - Notify on change

### ISSUE THREAD

**Issue by Quantas** [Quantas/shark-ng-table#62]
When entering text into the filters, there's no notification that anything happened. Ideally, a screenreader user should be notified that the data table has been updated due to the filtering. This feels like a particularly important notification to me; hence, the bug status because it's such a powerful way for a visual user to narrow down the rows. We want screenreader users to be able to achieve this understanding when filtering as well
